notebooks/0. Complete solution.ipynb

```diff
@@ -105,6 +105,26 @@
     "bgt_building_fuser = fusion.BGTBuildingFuser(Labels.BUILDING, building_offset=0.25, bgt_file=bgt_building_file, ahn_reader=ahn_reader)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "embedded-anthropology",
+   "metadata": {},
+   "source": [
+    "### Road segments\n",
+    "TODO"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fancy-legislation",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Road fuser using BGT road polygon data.\n",
+    "road_sidewalk_fuser = fusion.BGTRoadFuser(Labels.ROAD, bgt_file=bgt_road_file)"
```

Comment:
Dus nu labellen we zowel wegdelen als voetpaden als "ROAD"? Niet beter twee classes van maken?

Reply:
Wegdelen

Reply:
Misschien beter `road_part_fuser = fusion.BGT...`